src/components/site/SummaryList.js

```diff
@@ -1,20 +1,44 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 const SummaryList = props => {
-  const { items = [] } = props;
+  const { id, items = [] } = props;
   return (
     <div className="dg_summary-list">
       {items.map(item => (
-        <div key={item.key} className="dg_summary-list__item">
-          <i className={item.icon}></i>
-          <div>
-            <h3>{item.heading} </h3>
-            <div dangerouslySetInnerHTML={{ __html: item.content }} />
+        <div id={item.id} key={item.id} className="dg_summary-list__item">
+          <i
+            id={`summaryListItemIcon${item.id}`}
```

Comment:
just noticed id and key are everywhere, think you should only need this in the container div.

Something like is probably overkill

```jsx
<div
            id={`summaryListItem${item.id}`}
...
```

Reply:
also, not reflected in the markdown snippet

Reply:
id's removed